lib/js_routes/engine.rb

```diff
@@ -4,15 +4,27 @@ class JsRoutes
   SPROCKETS3 = Gem::Version.new(Sprockets::VERSION) >= Gem::Version.new('3.0.0')
   class Engine < ::Rails::Engine
 
-    initializer 'js-routes.dependent_on_routes', after: "sprockets.environment" do
+    if SPROCKETS3
+      initializer 'js-routes.dependent_on_routes', after: :engines_blank_point, before: :finisher_hook do
```

Comment:
@bogdan, we need `after: :engines_blank_point` in order to run this block after engines are initialized, `finisher_hook` is used as border as before this hook all initializations should be completed http://guides.rubyonrails.org/configuring.html#initializers